src/kernel-install/kernel-install.c

```diff
@@ -1088,12 +1123,14 @@ static int verb_remove(int argc, char *argv[], void *userdata) {
 static int verb_inspect(int argc, char *argv[], void *userdata) {
         Context *c = ASSERT_PTR(userdata);
         _cleanup_free_ char *joined = NULL;
+        const char *version;
         int r;
 
         c->action = ACTION_INSPECT;
+        version = argc > 1 ? argv[1] : NULL;
 
-        if (argc >= 2) {
-                r = context_set_kernel(c, argv[1]);
+        if (version) {
```

Comment:
Like `add`, maybe better to pick the current kernel image?

Reply:
i changed the code accordingly.